functions.Rmd

```diff
@@ -28,20 +28,23 @@ There are two options if the function name and definition can't fit on a single
     }
```

Comment:
Above, I find this a little confusing

```
Function-indent: place each argument on its own line
```

I'd argue that "place each argument on its own line" is actually what `Single-indent` is doing. i.e. I don't think of `a = "a long argument"` as being on its own line.

Maybe its 

```
Function-indent: add a line break after each `,`, and indent to match the opening `(` of `function`:
```